README.md

```diff
@@ -3,358 +3,84 @@
-
-```sh
-pip install autogen-agentchat~=0.2
-```
-
-### Will AutoGen Studio be supported in 0.4?
-
-Yes, this is on the [roadmap](#roadmap).
-Our current plan is to enable an implementation of AutoGen Studio
-on the AgentChat high level API which implements a set of agent functionalities
-(agents, teams, etc).
-
-### How do I migrate?
-
-For users familiar with AutoGen, the AgentChat library in 0.4 provides similar concepts.
-We are working on a migration guide.
-
-### Is 0.4 done?
-
-We are still actively developing AutoGen 0.4. One exciting new feature is the emergence of new SDKs for .NET. The python SDKs are further ahead at this time but our goal is to achieve parity. We aim to add additional languages in future releases.
-
-### What is happening next? When will this release be ready?
-
-We are still working on improving the documentation, samples, and enhancing the code. We are hoping to release before the end of the year when things are ready.
-
-### What is the history of this project?
-
-The rearchitecture of the framework started with multiple Microsoft teams coming together
-to address the gaps and learnings from AutoGen 0.2 - merging ideas from several predecessor projects.
-The team worked on this internally for some time to ensure alignment before moving work back to the open in October 2024.
-
-### What is the official channel for support?
-
-Use GitHub [Issues](https://github.com/microsoft/autogen/issues) for bug reports and feature requests.
-Use GitHub [Discussions](https://github.com/microsoft/autogen/discussions) for general questions and discussions.
-
-### Do you use Discord for communications?
-
-We are unable to use Discord for project discussions. Therefore, we request that all discussions take place on <https://github.com/microsoft/autogen/discussions/> going forward.
-
-### What about forks?
-
-<https://github.com/microsoft/autogen/> remains the only official repo for development and support of AutoGen.
-We are aware that there are thousands of forks of AutoGen, including many for personal development and startups building with or on top of the library. We are not involved with any of these forks and are not aware of any plans related to them.
-
-### What is the status of the license and open source?
+<div align="center">
 
-Our project remains fully open-source and accessible to everyone. We understand that some forks use different licenses to align with different interests. We will continue to use the most permissive license (MIT) for the project.
+|                      | [![Python](https://img.shields.io/badge/AutoGen-Python-blue?logo=python&logoColor=white)](./python)                                                                                     | [![.NET](https://img.shields.io/badge/AutoGen-.NET-green?logo=.net&logoColor=white)](./dotnet)              | [![Studio](https://img.shields.io/badge/AutoGen-Studio-purple?logo=visual-studio&logoColor=white)](./studio)              |
```

Comment:
Having a row for "Examples" would be great, examples are the best way to know how to use a library